Components/ObjectViewController.swift

```diff
@@ -21,6 +21,7 @@ class ObjectViewController: UIViewController {
             [PhoneNumberComponent(phoneNumber: "12345678", descriptionText: "Mobil", showNumberText: "Vis telefonnummer", accessibilityLabelPrefix: "Telefonnummer: ")],
             [CollapsableDescriptionComponent(text: attributedDescriptionText, titleShow: "+ Vis hele beskrivelsen", titleHide: "- Vis mindre")],
             [MessageButtonComponent(title: "Send melding", answerTime: "Svarer vanligvis innen 4 timer"), MessageButtonComponent(title: "Ring", answerTime: "Tar aldri telefonen")],
+            [PriceComponent(price: 1500000, currency: "kroner", accessibilityPrefix: "Pris: ")],
```

Comment:
I think here is better if you provide the locale as an optional argument, then you can use that to get the krone notation. https://stackoverflow.com/questions/4427680/iphone-how-to-get-local-currency-symbol-i-e-unstead-of-au

If the user doesn't provide a locale then you default to the current one.

I think the price should be a double, is not common to give price in Int.

Reply:
The currency string is to complement the accessibility label so that it reads properly. I'm not sure I can get it to read the number correctly if i don't have it.

 I have tried to use the `.locale` property on the formater, but the formatting gets a bit strange. Here are the different locale property formatting: 
- `formatter.locale = Locale(identifier: "no_NO")` looks like this: _kr 1 500 000,00_ 
- `formatter.locale = .current` looks like this: _NOK 1 500 000,00_

If I use the `.current` I don't need to provide the locale, but I think it looks weird.